src/components/Card.tsx

```diff
@@ -0,0 +1,54 @@
+import '../styles/Card.scss';
+import { Link } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+
+export interface CardProps {
+    imageSrc: string;
+    href: string;
+    title: string;
+    body: string;
+    showInitialText: boolean;
+    textHideable: boolean;
+    imageOnRight: boolean;
+    isTimelineCard: boolean;
+}
+
+export function Card({ imageSrc, href, title, body, showInitialText, textHideable, imageOnRight, isTimelineCard }: CardProps) {
+
+    const [textVisible, setTextVisible] = useState(showInitialText);
+
+    function changeToggleText(){
+        if(textHideable){
+            setTextVisible(!textVisible);
+        }  
+    }
+
+    const cardContent = (
+        <div className={`card-link card-image-on-${imageOnRight ? "right" : "left"}`}>
```

Comment:
I think we can extract the card class to this div here, then we can remove it from all the children

Reply:
I dont understand what you mean here.